### PR TITLE
Update README to provide a link to WSL instructions

### PR DESCRIPTION
**Pull request by lukeguild-quovo**
Adds instructions to help Windows users get Pattern up and running.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This is a reference application demonstrating an end-to-end [Plaid][plaid] integ
 
 ## Requirements
 
--   [Docker][docker] Version 2.0.0.3 (31259) or higher, installed, running, and signed in
+-   [Docker][docker] Version 2.0.0.3 (31259) or higher, installed, running, and signed in. If you're on **Windows**, check out [this link](https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly) to get set up in WSL.
```
Comment (evndean) [inline README.md]: 💯 gave the blog post the once-over, looks good to me.

**Comment by lukeguild-quovo** [comment]
@evndean ⚡️

**Comment by skylinebf25** [comment]
Thank you

**Review by Sb190NM — COMMENTED** [review]
32990403

**Review by Sb190NM — APPROVED** [review]

**Pull request merged**